src/helpers/setAuthorizationToken.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 const setAuthorizationToken = (token) => {
-  if (token) {
+  if (token !== '') {
```

Comment:
You sure about this ?
```javascript
undefined !== ''  => true
null !== '' => true
'string' !== '' => true
```
It's true for truthy values, null and undefined too.
Try to avoid the `!` operator. If you are using it make sure it works for all the conditions.